lib/fizzy/execute.cpp

```diff
@@ -6,6 +6,11 @@
 
 namespace fizzy
 {
+struct instance
```

Comment:
I think we should name types with CamelCase, because they conflict with natural var names. I'm ok with doing that already in this case: `Instance`.

Reply:
Why don't you make a PR which changes every existing one and we merge that first?

Reply:
This will probably take much longer.